modules/va_forms/lib/tasks/va_forms.rake

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+namespace :va_forms do
+  task fetch_latest: :environment do
+    VaForms::Form.refresh!
```

Comment:
Since we have to run this once a day would make more sense to make the whole thing a sidekiq job that runs daily and just call the job here to refresh the model.

Reply:
moved to sidekick worker and added entry to sidekick_scheduler